.github/workflows/test.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ master ]
+  workflow_call:
```

Comment:
What does this do?

Reply:
This makes `test.yml` into a callable unit.

You can see the other end of it here: https://github.com/nilenso/bugle-forms/pull/3/files#diff-755a4cb74ee9a85911356b3cab5844bfc09657ea29108b664aac29b907873f3bR5-R10